### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by Roveri91**
Set up Docker

- Create Dockerfile;
- Create Docker-compose
- Updated database.yml

Run `docker-compose build` and the `docker-compose up` on your terminal.

*_Make sure you have docker up and running _

### CODE REVIEW

**Review by srollins01 — DISMISSED** [review]
Was able to run the app successfully in Docker after creating and migrating the db.

File: Gemfile.lock [hunk 1]
```diff
@@ -228,6 +230,7 @@ GEM
 PLATFORMS
   arm64-darwin-22
   arm64-darwin-23
+  x86_64-linux
```
Comment (achpsident) [inline Gemfile.lock]: This should be probably be excluded, unless it's needed for the Dockerization.

File: Gemfile.lock [hunk 2]
```diff
@@ -128,6 +128,8 @@ GEM
     nio4r (2.6.1)
     nokogiri (1.15.5-arm64-darwin)
       racc (~> 1.4)
+    nokogiri (1.15.5-x86_64-linux)
```
Comment (achpsident) [inline Gemfile.lock]: See comment below.

File: config/database.yml [hunk 3]
```diff
@@ -21,9 +21,12 @@ default: &default
   # https://guides.rubyonrails.org/configuring.html#database-pooling
 
   # ADDED for Docker
-  host: postgresql
-  username: postgres
-  password: postgres
+  # host: postgresql
+  # username: postgres
+  # password: postgres
```
Comment (achpsident) [inline config/database.yml]: We don't need these any more. ;)

**Comment by achpsident** [comment]
Could you add the commands needed to get the Dockerized version up and running to the README? (The ones mentioned in your PR description, plus how to get a shell in to the Docker container to create/migrate the db)

File: config/database.yml (hunk 3 above)
Comment (Roveri91) [reply config/database.yml]: Remove unnecessary coments in database.yml and parts  in Gemfile.lock

**Comment by srollins01** [comment]
Reviewed the updates and everything looks good. The app runs without issues in Docker.

**Review by srollins01 — APPROVED** [review]

**Review by achpsident — APPROVED** [review]

**Pull request merged**